All EIPIP Meetings/Meeting 050.md

```diff
@@ -0,0 +1,251 @@
+
+**Shashank Yalamanchi**
+okay so so the new ips also include the current pull requests that haven't been merged would that be correct in the assumption
+
+**Micah Zoltu**
+yeah so any non-final EIP would probably fall under this rule if there's an EIP that has a whole lot of discussion already in a different place we it's probably fine but i would say we can just override when that happens like there might be one or two that are going to get grandfathered in and we can have one of the admins which would be manorai override in that case
+for the sake of the bot i would say have the bot very the sake of the bot have the bot be all be very strict so if if there is an EIP that is getting checked by the bot which should be any EIP that is currently has a pull request against it then have the bot validate that
+
+**Shashank Yalamanchi**
+okay so i think in one of the issue axic already mentioned a solution his suggestion was to ignore he said two rules one is basically ignore it for anything marked final superseded or abandoned and the second rule was add an explicit exclusion list for EIPs that is a way to disable this check for a set of EIPs and include those which are missing it currently but are merged 
+
+**Micah Zoltu**
+i'm also okay with either of those solutions okay those those solutions that alex proposed are more robust than the one i proposed the one i proposed is very easy so it depends if you if you feel you have the time and capacity to implement alex's solutions i do think those are better if we have to choose between doing nothing and doing my solution then i would pick my solution 
+
+**Shashank Yalamanchi**
+sure make sense okay that's there so matt there was this issue opened called remove updated will it be fine if i share my screen so it's easy for everyone? i kind of didn't understand this EIP should not be allowed in with updated so when i open this issue it points to this one
+
+**Micah Zoltu**
+yeah so there there is a very constrained set of fields that are allowed in the header and updated is not one of them and so i don't know if the validator currently validates that there are no extraneous fields i don't think it does but it should so the validator should when it's checking should make sure that there are no fields in the header besides the ones that are official fields
+
+**Shashank Yalamanchi**
+Got it. yeah i think that'll wait thank you
+
+**Pooja Ranjan**
+yeah i think that is from the one of the recent pull requests it's a new proposal fairly new proposal that had that additional fee and we were wondering like it would be nice to have the part validating it and i should not auto merge that
```

Comment:
```suggestion
yeah i think that is from the one of the recent pull requests it's a new proposal fairly new proposal that had that additional field and we were wondering like it would be nice to have the bot validating it and it should not auto merge that
```